modes/magit/evil-collection-magit.el

```diff
@@ -1,36 +1,52 @@
-;;; evil-collection-magit.el --- Bindings for `magit' -*- lexical-binding: t -*-
+;;; evil-collection-magit.el --- evil-based key bindings for magit
 
-;; Copyright (C) 2018 James Nguyen
+;; Copyright (C) 2015-2016 Justin Burkett
 
-;; Author: James Nguyen <james@jojojames.com>
-;; Maintainer: James Nguyen <james@jojojames.com>
+;; Author: Justin Burkett <justin@burkett.cc>
+;; Maintainer: Justin Burkett <justin@burkett.cc>
```

Comment:
Double checking you have commit access to evil-collection, I would've guessed members of emacs-evil have it.

Reply:
Yes, I think so